scripts/android-emulator-re-create.sh

```diff
@@ -1,33 +1,42 @@
 #!/bin/bash
 
-# This script will re-create Android emulators from 4.2 (android-17) to 7.1 (android-25)
+# This script will re-create Android emulators from 4.2 (android-17) to 8.1 (android-27)
 #
 # Notes:
 # This script will work on both Linux and macOS hosts!
 # Make sure emulator images exists (do not run before android-emulator-images-update.sh)!
 
+function recreate_emulator() {
+	name="$1"
+	image="$2"
+
+    echo no | $ANDROID_HOME/tools/bin/avdmanager create avd -n "${name}" -k "${image}" -c 12M -f
+}
+
 echo "Default Emulators"
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api17-Default -t android-17 --abi default/x86 -c 12M -f
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api18-Default -t android-18 --abi default/x86 -c 12M -f
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api19-Default -t android-19 --abi default/x86 -c 12M -f
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api21-Default -t android-21 --abi default/x86 -c 12M -f
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api22-Default -t android-22 --abi default/x86 -c 12M -f
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api23-Default -t android-23 --abi default/x86 -c 12M -f
-echo no | $ANDROID_HOME/tools/android create avd -n Emulator-Api24-Default -t android-24 --abi default/x86 -c 12M -f
+recreate_emulator Emulator-Api17-Default "system-images;android-17;default;x86"
+recreate_emulator Emulator-Api18-Default "system-images;android-18;default;x86"
+recreate_emulator Emulator-Api19-Default "system-images;android-19;default;x86"
+recreate_emulator Emulator-Api21-Default "system-images;android-21;default;x86"
+recreate_emulator Emulator-Api22-Default "system-images;android-22;default;x86"
+recreate_emulator Emulator-Api23-Default "system-images;android-23;default;x86"
+recreate_emulator Emulator-Api24-Default "system-images;android-24;default;x86"
 
 echo "ARM Emulators"
```

Comment:
I would keep only `default;x86` and `google_apis;x86 ` emulators and would remove `armeabi-v7a` from the script.

Reply:
ok

Reply:
Will remove arm64-v8a as well

Reply:
... and the mips one